TabloidMVC/Repositories/UserProfileRepository.cs

```diff
@@ -53,5 +55,92 @@ public UserProfile GetByEmail(string email)
                 }
             }
         }
+
+        public List <UserProfile> GetAllUsers()
+        {
+            using (var conn = Connection)
+            {
+                conn.Open();
+                using (var cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"
+                       SELECT u.id, u.FirstName, u.LastName, u.DisplayName, u.Email,
+                              u.CreateDateTime, u.ImageLocation, u.UserTypeId,
+                              ut.[Name] AS UserTypeName
+                         FROM UserProfile u
+                              LEFT JOIN UserType ut ON u.UserTypeId = ut.id
+                               ORDER BY u.DisplayName ASC;";
+
+                    var reader = cmd.ExecuteReader();
+
+                    var users = new List<UserProfile>();
+
+                    while (reader.Read())
+                    {
+                        users.Add(NewUserFromReader(reader));
+                    }
+
+                    reader.Close();
+
+                    return users;
+                }
+            }
+        }
+
+
+        public UserProfile GetUserById(int id)
```

Comment:
Is this method used anywhere in this PR? If not, it probably shouldn't be on this PR

Reply:
I am using this in my Controllers Index conditional